app/models/verification/dilution_plate/hamilton.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# Either 1 or 2 plates can be processed at a time.
+class Verification::DilutionPlate::Hamilton < Verification::DilutionPlate::Base
+  self.source_beds      = %w[SCRC4 SCRC5]
+  self.destination_beds = %w[DEST34 DEST35]
+end
```

Comment:
For some reason the javascript for the highlighting of the columns is not working. i.e. we have pairs of sources and destinations here and normally the beds highlight for each set. Related to use of SCRC and DEST rather than P names?